src/components/GridTemplates/DetailsPage1/DetailsPage1.stories.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file Details page template #1 story.
+ * @copyright IBM Security 2020
+ */
+
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { disableCentered, patterns } from '../../../../.storybook';
+import {
+  theme,
+  DetailsPage1,
+  DetailsPage1Main,
+  DetailsPage1Section,
+  DetailsPage1SectionTerm,
+  DetailsPage1SectionDefinition,
+  DetailsPage1Aside,
+} from '../../..';
+
+function DemoContent() {
+  return (
+    <div
+      style={{
+        backgroundColor: theme.interactive04,
+        minHeight: '240px',
```

Comment:
Let's use `@carbon/layout` for any of these pixel values where possible

Reply:
For demo content? We have a lot of story-specific style that do this, and Carbon does as well. Wouldn't that be overkill? It's kinda nice for a story to use the pixel value so that an end user can view this in the story source tab and immediately understand what the value is, without specialized knowledge of a Carbon-specific variable.

Reply:
Just revisiting this -- I still don't think it makes sense to use variables for mocked up / demo content.

Reply:
Hey @SimonFinney I'm seeing the reaction above so wanted to ask about it. Not sure what variable you want to see that comes out to 240px: https://www.carbondesignsystem.com/guidelines/spacing

And there are lots of other places in the storybook where we have demo pixel values step, to mock something up for the user. 🤔 Quick search in codebase's stories will get you these examples: `Accordion`, `CodeSnippet`, `Dropdown`, `FileUploader`, `FilterPanel`, `IconButtonBar`, `InlineLoading`, `MultiSelect`, `Nav`, `Notification`, `OverflowMenu`, `Pagination`, `PanelV2`, `Pill`, `ScrollGradient`, `Search`, `Select`, `Shell`, `SkeletonText`, `StructuredList`, `Tabs`, `Tile`, `UNSTABLE__Pagination`, and `Wizard`. I might have even missed one when copying the list, there are so many.

Reply:
No worries! If there isn't something readily-available from Carbon that makes sense, I'm happy to leave it as a pixel value - to your point, it seems like overkill unless there's something we can leverage for consistency